packages/playwright/src/matchers/expect.ts

```diff
@@ -61,6 +61,7 @@ import {
 import { zones } from 'playwright-core/lib/utils';
 import { TestInfoImpl } from '../worker/testInfo';
 import { ExpectError, isExpectError } from './matcherHint';
+import { randomUUID } from 'node:crypto';
```

Comment:
We are using createGuid() in our source base - Playwright was originally targeting Node 12, so consider using it for consistency, it has the same power.